docs/src/user_docs/assembly/cryptographic_operations.md

```diff
@@ -7,7 +7,7 @@ Miden assembly provides a set of instructions for performing common cryptographi
 | Instruction    | Stack_input     | Stack_output   | Notes                                      |
 | -------------- | --------------- | -------------- | ------------------------------------------ |
 | hash <br> - *(20 cycles)*  | [A, ...] | [B, ...] | $\{B\} \leftarrow hash(A)$ <BR> where, $hash()$ computes a 1-to-1 Rescue Prime Optimized hash. |
-| hperm  <br> - *(1 cycle)*      | [C, B, A, ...]  | [F, E, D, ...] | $\{D, E, F\} \leftarrow permute(A, B, C)$ <br> where, $permute()$ computes a Rescue Prime Optimized permutation. |
+| hperm  <br> - *(1 cycle)*      | [A, B, C ...]  | [A', B', C', ...] | $\{A', B', C'\} \leftarrow permute(C, B, A)$ <br> Performs a Rescue Prime Optimized permutation to the top 12 elements of the operand stack, where the first 8 elements are the rate (words A and B), the last 4 elements the capacity, (word C), and the result are the middle elements 4..8 (word B'). |
```

Comment:
I think the description is an improvement, although I would prefer "...and the output is the middle elements..." instead of "result". I prefer the previous alphabetization for the reasons mentioned in another comment.